### PR TITLE
Load initial resource templates into LD4P group

### LINKED ISSUE #67: Load initial resource templates into Sinopia server
When migrating (creating initial containers and ACLs), add initial set of resource templates to Sinopia server.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #67

Includes:
* Loop over resource template fixtures in `populateEmptyTrellis.js` and load each template into the LD4P group
* Make style of function definitions consistent in `populateEmptyTrellis.js`
* Add resource template fixtures and rename current fixture directory to be more general
* Allow platformdata docker-compose service to pass AWS credentials through
* Add new config option to set the default template group
* Update and clarify the README
* Add ignores to git and docker

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,10 @@
-npm-debug.log
-Dockerfile
+.circleci
```
Comment (mjgiarlo) [inline .dockerignore]: Also, sorted this file so it's easier to make sense of.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
-node_modules
-coverage
 .cognitoToken
+.env
```
Comment (mjgiarlo) [inline .gitignore]: Don't check in the .env file!

File: Dockerfile [hunk 3]
```diff
@@ -11,7 +11,4 @@ RUN /bin/bash -c 'chown -R circleci node_modules'
 
 USER circleci
 
-# Add babel-node to PATH
```
Comment (mjgiarlo) [inline Dockerfile]: I missed this in an earlier PR. This was only required when our `bin/` scripts directly invoked `babel-node`; now they invoke `babel-node` via `npx` which does not require PATH fidding.

File: src/populateEmptyTrellis.js [hunk 4]
```diff
@@ -26,39 +26,42 @@ import sleep from 'sleep'
...
+const templateDir = 'fixtures/resourceTemplates/'
+const templateContainerUrl = `${baseUrl}/repository/${config.get('resourceTemplateGroup')}`
 
-getToken()
+console.log('retrieving Cognito token')
 
-const cogTokenFile = config.get('cognitoTokenFile')
-const token = (fs.existsSync(cogTokenFile)) ?
-  fs.readFileSync(cogTokenFile, 'utf8').trim() :
+client.cognitoTokenToFile()
```
[the remark this answers is not in the document]
Comment (ndushay) [reply src/populateEmptyTrellis.js]: cool.  I kinda blocked on the async thing - didn't think simply to use sync versions of the calls.  D'oh.

**Comment by ndushay** [comment]
These could be a set of appropriate starting profiles, maybe:  https://github.com/LD4P/sinopia_sample_profiles/tree/master/profiles/v0.1.0

Certainly better than the ones we devs created/morphed just for exercising editor code.

**Comment by mjgiarlo** [comment]
@ndushay Thx! Will rebase on `master` once https://github.com/LD4P/sinopia_acl/pull/69 is merged and then clean up the commit message and GitHub description above.

**Comment by michelleif** [comment]
I would rather have the users (or me) upload templates manually instead of us prepopulating directly into a container, in which case we might miss any problems that arise when loading manually.

**Comment by mjgiarlo** [comment]
@michelleif OK, roger that!

**Pull request closed without merging**